### DISCUSSION TITLE
How highlight markdown inline code?

### DISCUSSION THREAD

**Discussion by pengqian089** [PrismJS/prism#3273] [Q&A] [answered]
I see from the official [documentation](https://prismjs.com/plugins/autoloader/) that using markdown will highlight inline code.
But my local code is not working.

``` html
<head>
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/prismjs@1.25.0/themes/prism-okaidia.min.css">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/prismjs@1.25.0/plugins/match-braces/prism-match-braces.min.css">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/prismjs@1.25.0/plugins/line-numbers/prism-line-numbers.min.css">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/prismjs@1.25.0/plugins/show-invisibles/prism-show-invisibles.min.css">
</head>

<body>
    
    <pre class="line-numbers language-markdown"><code>
`ShowPay()`


```csharp
using System;
using System.Threading.Tasks;
using Dpz.Core.WebMore.Models;
using Dpz.Core.WebMore.Service;
using Dpz.Core.WebMore.Shared;
using Microsoft.AspNetCore.Components;
using Microsoft.Extensions.Configuration;
using Microsoft.JSInterop;
using MudBlazor;```
</code></pre>


    <script src="https://cdn.jsdelivr.net/npm/prismjs@1.25.0/prism.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/prismjs@1.25.0/components.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/prismjs@1.25.0/plugins/autoloader/prism-autoloader.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/prismjs@1.25.0/plugins/match-braces/prism-match-braces.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/prismjs@1.25.0/plugins/line-numbers/prism-line-numbers.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/prismjs@1.25.0/plugins/show-language/prism-show-language.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/prismjs@1.25.0/plugins/show-invisibles/prism-show-invisibles.min.js"></script>    
    <script src="https://cdn.jsdelivr.net/npm/prismjs@1.25.0/plugins/copy-to-clipboard/prism-copy-to-clipboard.min.js"></script>
</body>
```

Execute `Prism.highlightAll()`,result:
![PrismJS-highlight-markdown](https://user-images.githubusercontent.com/13491546/144761358-219eb6fe-c6b7-4ce6-9490-5007f0c8fc84.png)

What should I do to highlight this C# code?

**Comment by RunDevelopment** [comment, marked as answer]
I looked into it, and it doesn't work for 2 reasons:

1. The easy: Your code block is invalid. The closing backticks have to be on a separate line.
2. The hard: Markdowns code highlighting seems to be incompatible with Show invisibles.

   I theorized that this might become a problem (#1609) but it never happened until now.

Not loading Show Invisibles makes Markdown's code highlighting work again, but that might not be an acceptable workaround for you.

Until #1609 is fixed, there is nothing we can do for you, unfortunately.

**Reply by pengqian089** [reply]
Thank you for your answer.
The closing backticks  is not on a separate line because github truncates the code block.In my local code closing backticks is on a separate line.
After I tried not loading Show Invisibles, it did work as expected.

Thank you again for your answer.
